spec/support/fill_in_the_form_steps.rb

```diff
@@ -116,6 +138,16 @@ def and_has_given_their_contact_details
     end
   end
 
+  def and_has_given_no_contact_details
+    expect(page).to have_content(I18n.t("coronavirus_form.questions.contact_details.title"))
+    within find(".govuk-main-wrapper") do
+      fill_in "phone_number_calls", with: ""
+      fill_in "phone_number_texts", with: ""
+      fill_in "email", with: ""
```

Comment:
Can these lines be removed? We want to test users only clicking the start button if possible.